Molyktech/Property-Pro-Lite
# #167157168 add a reset password endpoints

### What does this PR do?
Add a reset password endpoint 

#### Description of Task to be completed?
* Create a reset password route
* Add a reset password function to the User controller 
* Install nodemailer
* Configure nodemailer to send mail with password to the client

#### How should this be manually tested?
``clone the repo``
``cd into the directory Property-Pro-Lite``
``run npm install``
``run npm start``
``using POSTMAN, test by making a POST request to 'http://localhost:8000/api/v1/auth/:useremail/reset_password``
`` with an empty body or ``
`` { password: oldpassword,
     new_password: newPassword}``


make sure the header is set to:
 `` key: Content-Type ``
``value: application/json ``


#### What are the relevant pivotal tracker stories?
[#167157168] (https://www.pivotaltracker.com/story/show/167157168 )
